homeassistant/components/lutron_caseta.py

```diff
@@ -0,0 +1,109 @@
+"""
+Component for interacting with a Lutron Caseta system.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/lutron_caseta/
+"""
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (CONF_HOST,
+                                 CONF_USERNAME,
+                                 CONF_PASSWORD)
+from homeassistant.helpers import discovery
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['https://github.com/gurumitts/'
+                'pylutron-caseta/archive/v0.2.0.zip#'
+                'pylutron-caseta==v0.2.0', 'paramiko==2.1.2']
```

Comment:
I am having a bit of issue with this.  It is a dependency of the pylutron-caseta lib.  I thought that paramiko would be automatically installed as well but this is not the case.  I get import failures at run-time if I do not specify paramiko here.  Can you think of anything I should look into?